jenkins/pipelines/cd/tiup/prometheus-tiup-mirrior-update-test.groovy

```diff
@@ -122,6 +122,9 @@ node("build_go1130") {
                 checkoutTiCS(tag)
             }
         }
+        if (RELEASE_TAG >="v5.3.0" || RELEASE_TAG =="nightly" ) {
```

Comment:
this change also needed in `jenkins/pipelines/cd/tiup/grafana-tiup-mirror-update-test-hotfix.groovy`,it's our pre release pipeline.